consoles/sshXtermIPMI.pm

```diff
@@ -52,6 +54,12 @@ sub reset {
     return;
 }
 
+sub disable {
+    my ($self) = @_;
+    bmwqemu::diag "Try to deactivate IPMI SOL during disable";
+    eval { $self->backend->ipmitool("sol deactivate"); };
```

Comment:
this seriously looks like something you can put in a shared method

Reply:
> this seriously looks like something you can put in a shared method

Thanks for your comments!
Do you mean I need merge the two "eval" to one function?
Thank you again!

Reply:
Isn't `reset` already doing that? Why not call it here?

Reply:
I'm also wondering whether the error handling can be improved. I get that you want to ignore the case when there was nothing to deactivate but this way it silently ignores all possible errors.

And yes, `reset` is already doing that. Is the check for `$self->{activated}` the problem there?

Reply:
> Isn't `reset` already doing that? Why not call it here?

Thanks for your comments! I updated code to use reset instead of calling backend.

Reply:
> I'm also wondering whether the error handling can be improved. I get that you want to ignore the case when there was nothing to deactivate but this way it silently ignores all possible errors.
> 
> And yes, `reset` is already doing that. Is the check for `$self->{activated}` the problem there?

Thanks for your comments!

Adding eval for deactivate SOL in function activate is for safety and avoiding SOL active issue causes the test failure.
Using $self->reset instead of calling backend makes sense and resubmitted PR.